vllm/worker/hpu_model_runner.py

```diff
@@ -245,6 +245,8 @@ def find_rope_layer(parent, path):
     # Return the result if found, otherwise None
     return path_to_rope
 
+def is_gaudi2() -> bool:
+    return htorch.hpu.get_device_name() == "GAUDI2"
```

Comment:
from vllm_hpu_extension.ops import is_hpu_gaudi2

Reply:
Hi @xinyu-intel Thank you for the advice. I have updated the code here. Thanks!